ibmcloudshellv1/ibm_cloud_shell_v1_examples_test.go

```diff
@@ -148,10 +148,6 @@ var _ = Describe(`IBMCloudShellV1 Examples Tests`, func() {
 					Enabled: core.BoolPtr(true),
 					Key:     core.StringPtr("eu-de"),
 				},
-				{
-					Enabled: core.BoolPtr(true),
-					Key:     core.StringPtr("jp-tok"),
-				},
```

Comment:
Do you intend to remove this? Just asking because as far as I can see, the API hasn't changed.

Reply:
Ok, thanks!